modules/structs/org.go

```diff
@@ -4,16 +4,27 @@
 
 package structs
 
+import "time"
```

Comment:
```suggestion
import (
	"time"
)
```

Reply:
Going to stop letting my IDE do the imports....will turn that off. 😡